spec/mongo/collection_spec.rb

```diff
@@ -3386,6 +3386,7 @@ def generate
       around(:each) do |spec|
         authorized_client[:validating,
                           :validator => { :a => { '$exists' => true } }].tap do |c|
+          c.drop
```

Comment:
After the changes made to reconnect the clients, this below line was raising an error due to the collection already being created